### ISSUE TITLE
Display the program name in USAGE

### ISSUE THREAD

**Issue by pvdlg** [mattallty/Caporal.js#85]
`package.json` [allow to define a command name](https://docs.npmjs.com/files/package.json#bin) to run the cli, which might be different than the filename in which its implemented.

There is an undocumented `name(string)` function that allow to set the program name. The name set with this function is used to display the fist line of the help message but in the USAGE part.

For example with the following code:
```
// mycli.js
 prog.name('my-cly')
    .version('1.0.0')
    .description('My cli description')
```
The help will be:
```
   my-cli 1.0.0 - My cli description
     
   USAGE

     mycli.js 
```
But it should be:
```
   my-cli 1.0.0 -My cli description
     
   USAGE

     my-cli
```

**Comment by thecodejunkie** [comment]
@pvdlg @pauloborges I just submitted #97 hopefully we can get access to this asap 😄

**Comment by mattallty** [comment]
Closed by #97 thanks to @thecodejunkie
